--- scripts/ci/do_not_merge.py
+++ scripts/ci/do_not_merge.py
@@ -15,10 +15,13 @@


 def print_rate_limit(gh, org):
-    response = gh.get_organization(org)
-    for header, value in response.raw_headers.items():
-        if header.startswith("x-ratelimit"):
-            print(f"{header}: {value}")
+    try:
+        response = gh.get_organization(org)
+        for header, value in response.raw_headers.items():
+            if header.startswith("x-ratelimit"):
+                print(f"{header}: {value}")
+    except github.GithubException:
+        pass


 def parse_args(argv):
